### PR TITLE
Add support for custom user ssh configs

### PR DESCRIPTION
**Pull request by bbinet**

### CODE REVIEW

File: openssh/files/ssh_config [hunk 1]
```diff
@@ -1,29 +1,71 @@
 {%- from "openssh/map.jinja" import client with context %}
 {%- from "linux/map.jinja" import network with context %}
```
Comment (fpytloun) [inline openssh/files/ssh_config]: Is this still needed? I would prefer removing cross-formula dependencies that are not needed anymore.

File: openssh/files/ssh_config [hunk 2]
```diff
@@ -1,29 +1,71 @@
 {%- from "openssh/map.jinja" import client with context %}
```
Comment (fpytloun) [inline openssh/files/ssh_config]: Use `-%}` here to strip empty lines at the beginning of file.

File: openssh/files/ssh_config (hunk 1 above)
Comment (bbinet) [reply openssh/files/ssh_config]: network is still used:
- https://github.com/salt-formulas/salt-formula-openssh/pull/10/files#diff-41b97c7794214fea83bdeef3cf284554R9
- https://github.com/salt-formulas/salt-formula-openssh/pull/10/files#diff-4f7b47e0118a01d2898261d8a8538c72L19

File: openssh/files/ssh_config (hunk 2 above)
Comment (bbinet) [reply openssh/files/ssh_config]: Done in 7de3458.

**Comment by fpytloun** [comment]
Great, merged :-)

**Pull request merged**